core/src/epicli/data/common/ansible/playbooks/roles/kibana/templates/kibana.yml.j2

```diff
@@ -32,7 +32,11 @@ elasticsearch.hosts:
 
 elasticsearch.ssl.verificationMode: none
 elasticsearch.username: kibanaserver
-elasticsearch.password: kibanaserver
```

Comment:
Shouldn't we make configurable also login/username?

Reply:
We should, but this should be connected to functionality of adding/removing users. Right now we don't have such functionality.